lib/rubocop/cop/mixin/multiline_expression_indentation.rb

```diff
@@ -222,6 +239,15 @@ def inside_arg_list_parentheses?(node, ancestor)
           node.source_range.end_pos < ancestor.loc.end.end_pos
       end
 
+      # Returns true if `node` is a conditional whose `body` and `condition`
+      # begin on the same line.
+      def postfix_conditional?(node)
+        node.if_type? &&
+          node.body &&
```

Comment:
There's a node extension that provides `#modifier_form?`, which matches:

```
<expression> <keyword> <predicate>
```

Could that be used here?

The current implementation looks like it has some false positives, e.g.

```ruby
if foo then bar end

if foo; bar; end
```